server/mergin/sync/models.py

```diff
@@ -365,7 +365,7 @@ class ProjectAccessDetail:
     role: str
     username: str
     name: Optional[str]
-    project_permission: str
+    project_permission: ProjectRole
```

Comment:
do we need this?

Reply:
i thought it is more descriptive than general string, do you want to put it back?

Reply:
I'd stick to ProjectRole, better than using random string, just wondering if something did not break in code - could you check usage?

Reply:
It's not ProjectRole.... we don't have enum for ProjectPermission (read/write ... )